tests/data/axis/convert_axis.py

```diff
@@ -43,8 +43,8 @@ def test_w1_wa():
     assert data.wa.units == "wn"
     data.wa.convert("eV")
     assert data.wa.units == "eV"
-    assert np.isclose(data.wa.max(), 1.5802564757220569)
-    assert np.isclose(data.wa.min(), 0.6726385958618104)
+    assert np.isclose(data.wa.max(), 1.5800551001941774)
+    assert np.isclose(data.wa.min(), 0.6725528801867734)
```

Comment:
These constants were computed _less_ precisely with our own units library.

The same dataset (and thus constants checked) was used when converting axes, data object, and dataset directly)